src/_langs/en/fundamentals/discovery-and-distribution/avoid-mixed-content/what-is-mixed-content.markdown

```diff
@@ -1,15 +1,14 @@
 ---
 layout: article
-published: false
```

Comment:
Does this work like I think it does? Would removing this hear but leaving Published:false in the index.markdown make it visible in prod but unlisted?